### PR TITLE
Implement dynamic metric

### PR DESCRIPTION
**Pull request by brandboat**

### CODE REVIEW

File: accio-sqlrewrite/src/main/java/io/accio/sqlrewrite/analyzer/StatementAnalyzer.java [hunk 1]
```diff
@@ -158,21 +160,28 @@ protected Scope visitTable(Table node, Optional<Scope> scope)
...
-                        .orElseGet(ImmutableList::of)
-                        .stream()
-                        .map(column ->
-                                Field.builder()
-                                        .modelName(tableName)
-                                        .columnName(column.getName())
-                                        .name(column.getName())
-                                        .build())
-                        .collect(toImmutableList());
+                List<Column> columns = new ArrayList<>();
```
Comment (goldmedal) [inline accio-sqlrewrite/src/main/java/io/accio/sqlrewrite/analyzer/StatementAnalyzer.java:163]:
```suggestion
                List<Column> columns = ImmutableList.of();
```

**Review by goldmedal — APPROVED** [review]
Minor comments. Others LGTM.

File: accio-sqlrewrite/src/main/java/io/accio/sqlrewrite/RelationableSqlRender.java [hunk 2]
```diff
@@ -30,12 +30,9 @@
 import java.util.Optional;
 import java.util.Set;
 
-import static io.accio.sqlrewrite.Utils.parseQuery;
-import static java.lang.String.format;
-import static java.lang.String.join;
 import static java.util.Objects.requireNonNull;
-import static java.util.stream.Collectors.joining;
 
+// TODO: Turn this into interface
```
[the remark this answers is not in the document]
Comment (brandboat) [reply accio-sqlrewrite/src/main/java/io/accio/sqlrewrite/RelationableSqlRender.java:35]: Anyway, I'm not going to do this right now.

File: accio-sqlrewrite/src/main/java/io/accio/sqlrewrite/analyzer/StatementAnalyzer.java (hunk 1 above)
Comment (brandboat) [reply accio-sqlrewrite/src/main/java/io/accio/sqlrewrite/analyzer/StatementAnalyzer.java]: Already addressed comments in the latest commit.

**Pull request merged**